draft-irtf-cfrg-opaque.md

```diff
@@ -1411,6 +1307,11 @@ Steps:
 
 #### Shared Secret Derivation
 
+Each party generates own private and public keyshares in each session using the following function.
+
+- GenerateAuthKeyPair(): Return a randomly generated private and public key pair. This can be
```

Comment:
Why is this here?

Reply:
Not sure why it ended up here. I want to define it in the AKE section, since this function is only used there, for the client and server to get their DH keys. I'll revert and work over that in a different PR